fixed a bug in decompose_mesh_MPI introduced when I added support for QKappa; removed an old sentence in the users manual that said that the internal mesher did not support PMLs (it now does)

komatits opened the pull request.

buildbot-princeton commented:
The files changed require a buildbot test. Buildbot has started.

luet commented:
It’s taking a long time. Let me look into it.

From: buildbot-princeton <notifications@github.com>
Reply-To: geodynamics/specfem3d <reply@reply.github.com>
Date: Thursday, October 25, 2018 at 5:02 PM
To: geodynamics/specfem3d <specfem3d@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Subject: Re: [geodynamics/specfem3d] fixed a bug in decompose_mesh_MPI introduced when I added support for QKappa; removed an old sentence in the users manual that said that the internal mesher did not support PMLs (it now does) (#1305)


The files changed require a buildbot test. Buildbot has started.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/geodynamics/specfem3d/pull/1305#issuecomment-433204551>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AFY0TEESoEx4aam6SX7j25NWM0k-TzrYks5uoibZgaJpZM4X7GZO>.

komatits commented:
Thanks!

Dimitri.

On 10/25/18 11:24 PM, David Luet wrote:
> ok. it was stuck but it’s really running now.
> 
> From: buildbot-princeton <notifications@github.com>
> Reply-To: geodynamics/specfem3d <reply@reply.github.com>
> Date: Thursday, October 25, 2018 at 5:02 PM
> To: geodynamics/specfem3d <specfem3d@noreply.github.com>
> Cc: Subscribed <subscribed@noreply.github.com>
> Subject: Re: [geodynamics/specfem3d] fixed a bug in decompose_mesh_MPI 
> introduced when I added support for QKappa; removed an old sentence in 
> the users manual that said that the internal mesher did not support PMLs 
> (it now does) (#1305)
> 
> 
> The files changed require a buildbot test. Buildbot has started.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on 
> GitHub<https://github.com/geodynamics/specfem3d/pull/1305#issuecomment-433204551>, 
> or mute the 
> thread<https://github.com/notifications/unsubscribe-auth/AFY0TEESoEx4aam6SX7j25NWM0k-TzrYks5uoibZgaJpZM4X7GZO>. 
> 
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/geodynamics/specfem3d/pull/1305#issuecomment-433211149>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFjDKX3C-A1hhlXseKk9o87Z15zoAIzVks5uoiwjgaJpZM4X7GZO>.
> 

-- 
Dimitri Komatitsch, CNRS Research Director (DR CNRS)
Laboratory of Mechanics and Acoustics, Marseille, France
http://komatitsch.free.fr

buildbot-princeton commented:
The files changed require a buildbot test. Buildbot has started.

buildbot-princeton commented:
Your changes passed the buildbot test. See [build 843](http://specfem3d-buildbot.princeton.edu/builders/specfem3d_builder/builds/843).

komatits commented:
accept

On 10/26/18 12:11 AM, buildbot-princeton wrote:
> Your changes passed the buildbot test. See build 843 
> <http://specfem3d-buildbot.princeton.edu/builders/specfem3d_builder/builds/843>.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/geodynamics/specfem3d/pull/1305#issuecomment-433222872>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFjDKYusEot9UkCIQpSZ21fd2tQweSgEks5uojb2gaJpZM4X7GZO>.
> 

-- 
Dimitri Komatitsch, CNRS Research Director (DR CNRS)
Laboratory of Mechanics and Acoustics, Marseille, France
http://komatitsch.free.fr
